Sources/FileSystem/Model/App.swift

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+import Model
+
+struct App: Codable {
+
+    let id: String
```

Comment:
I wonder here why don't make id as `var` as well?

Reply:
I'll amend it